### PR TITLE
Use Cloudsmith Apt repo proxy to install workstation tooling

### PR DESCRIPTION
**Pull request by christophermaier**
This simplifies our setup script by offloading the upstream repository
configuration to our proxy.

Signed-off-by: Christopher Maier <chris@graplsecurity.com>

### CODE REVIEW

File: etc/chromeos/lib/installs.sh [hunk 1]
```diff
@@ -268,24 +279,16 @@ install_hashicorp_tools() {
     echo_banner "Installing hashicorp tools: consul nomad packer"
 
     # Set specific versions since we're enabling the hashicorp test repo
-    CONSUL_VERSION="1.12.0"
+    CONSUL_VERSION="1.12.0-1"
     NOMAD_VERSION="1.3.0-1~beta.1"
     PACKER_VERSION="1.8.0"
     VAULT_VERSION="1.10.0"
```
Comment (wimax-grapl) [inline etc/chromeos/lib/installs.sh:285]: see https://github.com/grapl-security/grapl/pull/1695

Comment (twunderlich-grapl) [reply etc/chromeos/lib/installs.sh:285]: @christophermaier I just merged #1695 in (I had forgotten I hadn't done that yet). That should fix the versions once you've merged this in and dealt with the conflict.

Comment (christophermaier) [reply etc/chromeos/lib/installs.sh:285]: Ah, cool 👍

**Review by twunderlich-grapl — APPROVED** [review]
Thanks for doing this @christophermaier

**Pull request merged**